docs/osquery/osquery-placeholder-fields.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[osquery-placeholder-fields]]
+== Use placeholder fields in Osquery queries 
+
+Instead of hard-coding alert and event values into Osquery queries, you can use placeholder fields to dynamically pass this data into queries. Placeholder fields function like replaceable parameters. You can use placeholder fields to build flexible and re-usable queries. 
+
+Placeholder fields work in single queries or query packs. They are also supported in the following features:
+
+* <<alerts-run-osquery,Live queries>>
+* <<osquery-response-action,Osquery Response Action>>
++
+NOTE: Placeholder fields within an Osquery Response Action query will only work if the rule has generated alerts.
```

Comment:
Not sure I understand the purpose of this note. I could be missing something, but why would someone expect a response action to run if it's parent rule hadn't generated an alert?